torch/ao/ns/_numeric_suite_fx.py

```diff
@@ -862,7 +870,7 @@ def loggers_set_save_activations(
         if isinstance(child, OutputLogger):
             child.save_activations = save_activations
 
-def convert_n_shadows_model(model: GraphModule) -> GraphModule:
+def convert_n_shadows_model(model: GraphModule, exposed_convert_function: Any=None, exposed_convert_kwargs: Any=None) -> GraphModule:
```

Comment:
can we use `Callable` and `Dict[str, Any]` as the respective types?

Reply:
fixed